plugin/tagbar.vim

```diff
@@ -95,31 +95,31 @@ endif
 
 function! s:setup_keymaps() abort
     let keymaps = [
-        \ ['jump',          '<CR>'],
-        \ ['preview',       'p'],
-        \ ['previewwin',    'P'],
-        \ ['nexttag',       '<C-N>'],
-        \ ['prevtag',       '<C-P>'],
-        \ ['showproto',     '<Space>'],
+        \ ['jump',          '<cr>'],
+        \ ['preview',       'P'],
+        \ ['previewwin',    'W'],
+        \ ['nexttag',       '<C-J>'],
+        \ ['prevtag',       '<C-K>'],
+        \ ['showproto',     'I'],
         \ ['hidenonpublic', 'v'],
         \
-        \ ['openfold',      ['+', '<kPlus>', 'zo']],
-        \ ['closefold',     ['-', '<kMinus>', 'zc']],
-        \ ['togglefold',    ['o', 'za']],
-        \ ['openallfolds',  ['*', '<kMultiply>', 'zR']],
-        \ ['closeallfolds', ['=', 'zM']],
-        \ ['incrementfolds',  ['zr']],
-        \ ['decrementfolds',  ['zm']],
-        \ ['nextfold',      'zj'],
-        \ ['prevfold',      'zk'],
+        \ ['openfold',       '+'],
```

Comment:
Why did you drop all the standard keymaps (that match vim's defaults?)

Reply:
Sorry, the second commit was originally only for my local use. As I was not familiar with the function of pull requst, the second commit error was more closely related to pull request. I originally just wanted to submit the first pull request.

Reply:
Okay. You can remove it by resetting your branch to the correct commit and force-pushing. Keep in mind Github will keep this PR up to date with your branch, so you should do the work for the PR in a different branch than the one you are using. This is why Github shows you a big error message when you open PRs against your master branch saying that is an anti-pattern!


My suggestion would be to start a second branch for your own use, then tweak this one to remove that commit (and also address other issues brought up in review here.